src/www/index.html

```diff
@@ -156,7 +156,7 @@
   </style>
 </head>
 
-<body>
+<body onLoad="document.getElementById('email').focus();">
```

Comment:
Add this to `form.js`
https://github.com/fossasia/open-event-webapp/blob/development/src/www/js/form.js#L32

Reply:
Thanks @geekyd 
Done :+1: